src/main/java/printHelper.java

```diff
@@ -0,0 +1,63 @@
+import java.util.Random;
+
+public class printHelper {
+    /**
+     * Prints "*" as line-separator
+     */
+    public static void printLine() {
+        final int CHARS_IN_LINE = 40;
+        for (int i = 0; i < CHARS_IN_LINE; i++) {
```

Comment:
Good format in for loops with only one statement in the loop body and correct curly brackets